### PR TITLE
[risk=low][RW-7616] Update DUCC to v4 in UI

### PR DESCRIPTION
**Pull request by jmthibault79**
Note: v3 is still acceptable.  We will not be removing any user's access for this change.

The new version of the DUCC is from [here](https://docs.google.com/document/d/1UzNstlXzcN0nEDdhA_ETC249Wr61zM91jRnCEsOX3ok/edit#) (note that they call it v3)

Tested locally by visiting the DUCC manually at http://localhost:4200/data-code-of-conduct.  After accepting this DUCC, it recorded a v4 in the DB for my user, which maintained access.  My user who had signed only v3 also maintained access.  

The DUCC HTML file was generated using this process: https://github.com/all-of-us/workbench/wiki/Updating-the-DUCC-and-TOS

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/main/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/public/data-user-code-of-conduct-v4.fld/colorschememapping.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```
Comment (jmthibault79) [inline ui/public/data-user-code-of-conduct-v4.fld/colorschememapping.xml:1]: The files in the FLD directory and the HTML file were created through this process:
* save GoogleDoc as .docx
* open in Microsoft Word
* save as HTML

See also https://precisionmedicineinitiative.atlassian.net/browse/RW-4977

File: ui/src/app/pages/profile/data-user-code-of-conduct.tsx [hunk 2]
```diff
@@ -253,6 +260,18 @@ export const DataUserCodeOfConduct = fp.flow(
                   participants with meaningful information about the research
                   being conducted.
                 </InitialsAgreement>
+                <InitialsAgreement
```
Comment (jmthibault79) [inline ui/src/app/pages/profile/data-user-code-of-conduct.tsx:263]: The new DUCC introduces a third initial box

File: ui/src/app/components/bug-report.spec.tsx [hunk 3]
```diff
@@ -3,14 +3,14 @@ import * as React from 'react';
 
 import { BugReportModal } from './bug-report';
 import { registerApiClient } from 'app/services/swagger-fetch-clients';
-import { Profile, ProfileApi } from 'generated/fetch';
```
Comment (jmthibault79) [inline ui/src/app/components/bug-report.spec.tsx:6]: a few opportunistic cleanups as I saw some antipatterns

File: ui/src/app/pages/profile/data-user-code-of-conduct.spec.tsx [hunk 4]
```diff
@@ -141,6 +142,6 @@ describe('DataUserCodeOfConduct', () => {
       wrapper.find('[data-test-id="submit-ducc-button"]').prop('disabled')
     ).toBeFalsy();
     wrapper.find('[data-test-id="submit-ducc-button"]').simulate('click');
-    expect(spy).toHaveBeenCalledWith(3, 'XX'); // duccSignedVersion
```
Comment (jmthibault79) [inline ui/src/app/pages/profile/data-user-code-of-conduct.spec.tsx:144]: well obviously this one failed

**Review by freemabd — APPROVED** [review]
LGTM

**Pull request merged**